rubocop-factory_bot.gemspec

```diff
@@ -1,25 +1,20 @@
 # frozen_string_literal: true
 
 $LOAD_PATH.unshift File.expand_path('lib', __dir__)
-require 'rubocop/rspec/version'
+require 'rubocop/factory_bot/version'
 
 Gem::Specification.new do |spec|
-  spec.name = 'rubocop-rspec'
-  spec.summary = 'Code style checking for RSpec files'
+  spec.name = 'rubocop-factory_bot'
+  spec.summary = 'Code style checking for FactoryBot files'
   spec.description = <<-DESCRIPTION
-    Code style checking for RSpec files.
+    Code style checking for FactoryBot files.
     A plugin for the RuboCop code style enforcing & linting tool.
   DESCRIPTION
-  spec.homepage = 'https://github.com/rubocop/rubocop-rspec'
+  spec.homepage = 'https://github.com/rubocop/rubocop-factory_bot'
   spec.authors = ['John Backus', 'Ian MacLeod', 'Nils Gemeinhardt']
```

Comment:
memo) The author has not changed from RuboCop RSpec, but who is appropriate?

Reply:
I suggest blaming the lib/cops folder and seeing who the committers were.
For capybara cops that was almost exclusively you, @ydah.

Side note: @r7kamura you seem to have in-depth knowledge of FB, would you be actively contributing to this repo?

Reply:
Thank you for your efforts on this gem split 👍 
I don't have the confidence at this time to be involved in the ongoing maintenance of this gem, but I do have a few new cop ideas I would like to add.

Reply:
I was looking at the results of git blame and it did not seem that many FactoryBot department were committed by any particular contributor.

### Number of PR
```
pirj 4
Darhazer 4
abrom 4
composerinteralia 3
ydah 3
r7kamura 2
bquorning 2
jfragoulis 2
```

### Author of the cop.
```
FactoryBot/FactoryNameStyle: ydah
FactoryBot/ConsistentParenthesesStyle: Liberatys
FactoryBot/SyntaxMethods: leoarnold
FactoryBot/FactoryClassName: jfragoulis
FactoryBot/AttributeDefinedStatically: composerinteralia, seanpdoyle
FactoryBot/CreateList: Darhazer
FactoryBot/StaticAttributeDefinedDynamically: abrom
```